cmd/provider.go

```diff
@@ -4,12 +4,15 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var providerHelpMsg = "Provider command that unifies provider subcommands."
```

Comment:
"top-level command to interact with providers"

Reply:
Updated.